testutils/ttn.py

```diff
@@ -50,6 +50,8 @@ def __enter__(self):
         try:
             password = get_required_envvar("TTN_DL_KEY")
         except RuntimeError:
+            # Deprecated, keep old name for retro-compatibility with previous
+            # release
```

Comment:
It's less about the previous release, but the bootstrapping of having `LORAWAN_DL_KEY` still [in `.github/workflows/release-test.yml` over at RIOT-OS/RIOT](https://github.com/RIOT-OS/RIOT/blob/master/.github/workflows/release-test.yml#L133) while here we already use `TTN_DL_KEY`. Once this PR is merged and we renamed `LORAWAN_DL_KEY` to `TTN_DL_KEY` over at RIOT-OS/RIOT, these lines can be deleted. So please reword

Reply:
Then maybe just wait for that PR instead?

Reply:
Ahh wait there is not PR doing it yet, undestood

Reply:
better?

Reply:
Then that PR and the other need to be merged basically simultaneously, that was what I meant with bootstrapping problem ;-).

Reply:
> better?

Yepp :-)